pkg/pagerduty/mock/mock_service.go

```diff
@@ -65,11 +65,9 @@ func (mr *MockClientMockRecorder) GetIntegrationKey(data interface{}) *gomock.Ca
 }
 
 // CreateService mocks base method
-func (m *MockClient) CreateService(data *pagerduty.Data) (string, error) {
+func (m *MockClient) CreateService(data *pagerduty0.Data) (string, error) {
 	m.ctrl.T.Helper()
 	ret := m.ctrl.Call(m, "CreateService", data)
-	data.ServiceID = "XYZ123"
```

Comment:
I'm not sure how this was generated previously but I ran `mockgen -source=service.go -destination=./mock/mock_service.go` which sortof matches the current mockfile. This change seems to be added directly to the generated files though so I have moved it to

Worth getting this generated by boilerplate but im not 100% up on that process